siac/hostcmd.go

```diff
@@ -117,16 +117,33 @@ sector may impact host revenue.`,
 // Prints info about the host and its storage folders.
 func hostcmd() {
 	hg := new(api.HostGET)
-	err := getAPI("/host", &hg)
+	err := getAPI("/host", hg)
 	if err != nil {
 		die("Could not fetch host settings:", err)
 	}
 	sg := new(api.StorageGET)
-	err = getAPI("/storage", &sg)
+	err = getAPI("/storage", sg)
 	if err != nil {
 		die("Could not fetch storage info:", err)
 	}
 
+	// Determine the competitive price string.
+	ah := new(api.ActiveHosts)
+	var competitivePrice string
+	err = getAPI("/renter/hosts/active?numhosts=24", ah)
+	if err != nil {
+		competitivePrice = "Unavailable"
+	} else if len(ah.Hosts) > 0 {
```

Comment:
if `err == nil` but `len(ah.Hosts) == 0`, this will result in `competitivePrice = ""`